lm_eval/evaluator.py

```diff
@@ -5,15 +5,16 @@
 import lm_eval.models
 import lm_eval.tasks
 import lm_eval.base
+from scripts.clean_training_data.contamination import get_train_overlap
 import numpy as np
 from lm_eval.utils import positional_deprecated
 
-
 @positional_deprecated
 def simple_evaluate(model, model_args=None, tasks=[],
                     num_fewshot=0, batch_size=None, device=None,
                     no_cache=False, limit=None, bootstrap_iters=100000,
-                    description_dict=None):
+                    description_dict=None, decontaminate=False, 
+                    ngrams_path=None, ngrams_n_size=None):
```

Comment:
nit: the ngrams arguments should be prefixed with `decontamination_` or something

Reply:
Counterpoint is very long parameters, but I'm not particularly fussed in changing it.

Reply:
maybe `decont_` as a compromise